docs/decisions/0002-footer-links.rst

```diff
@@ -0,0 +1,168 @@
+            ],
+        };
+
+        export default config;
+
+* At runtime, via ``MFE_CONFIG`` in LMS's Django settings:
+
+    .. code-block:: python
+
+        MFE_CONFIG = {
+            "FOOTER_LINKS": [
+                {"url": "https://google.com", "text": "Terms of service"},
+                {"url": "https://duckduckgo.com", "text": "Acceptable Use Policy"},
+                {"url": "https://openedx.org", "text": "Privacy Policy"},
+            ],
+        }
+
+In case the array is undefined or empty, we won't render any links (or it's
+container).
+
+.. _MFE's config mechanism: https://github.com/openedx/frontend-platform/blob/master/src/config.js
+
+Styles
+======
+
+Since the root of the footer component is rendered as a footer HTML element, it
+is unique enough to use ``footer`` as selector. To make it more convenient to
+style footer's children, we will add unique class names, which will simplify
+the SCSS selectors.
+
+These SCSS styles can be put in a custom `branding package`_ into
+``paragon/_overrides.scss``.
+
+.. _branding package: https://github.com/openedx/brand-openedx
+
+Consequences
+************
+
+.. This section describes the resulting context, after applying the decision. All consequences should be listed here, not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences, but all of them affect the team and project in the future.
+
+Positive
+========
+
+* There will be an option to make the MFE footer look similar to the legacy UI
+  footer, without having to create and maintain a fork.
+* Can add custom text/urls for links, unlike what branding v1 API allows.
+
+Negative
+========
+
+* There are already parameters that allow configuring the footer links for the
+  legacy UI, which means that configuration will be duplicated (potentially in
+  multiple places), in cases when the aim is to make the links in the legacy UI
+  and MFE footers that same.
+
+Rejected Alternatives
+*********************
+
+.. This section lists alternate options considered, described briefly, with pros and cons.
+
+React component props API
+=========================
+
+Instead of reading the configuration values in the component, we could be
+reading it somewhere else, and passing the values as props to the react
+component. We couldn't identify any pros to this approach, and the cons are the
+same as the cons from decoupling.
+
+Styling using CSS variables
+===========================
+
+Exposing styling customizations through the CSS variables would be similar to
+styling via CSS classes, and it would be better aligned with the current
+philosophy of Paragon, since it prefers CSS variables over custom CSS classes.
+
+The cons however, is that you can only expose one CSS attribute per variable.
+This means that we would either heavily limit the styling options, or would
+have to create many variables for each element and CSS attribute.
```

Comment:
I don't think we need to include this. It doesn't really compete with this ADR. You can still use CSS variables in the classes, and a future version of Paragon will do just that.